.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: Legacy Selenium Firefox
 on:
   push:
     branches: [master]
-  pull_request:
+  pull_request_target:
```

Comment:
Probably reverting your change might make it run again, isnt this fix supposed to be on uitestframework not on distro

Reply:
> isnt this fix supposed to be on uitestframework not on distro

Sure! @sherrif10 would you recommend shifting these changes to _contrib_ module to make this resolved

Reply:
You would probably need to create another pull request in contrib uitestframework i think , however looking forward to hear from cc @k-joseph  about this work around

Reply:
This event pull_request_target was introduced by GitHub recently in order to support the PR from forked repositories. Here is the [documentation ](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target).